_devconfig/index.md

```diff
@@ -1,12 +1,10 @@
 ---
 layout: default
-title: Introduction
+title: Introduction to PIV Developer Guides
 permalink: /devconfig/
 collection: devconfig
 ---
 
-## Introduction to PIV Developer Guides
-
```

Comment:
This is one of a few places where I ditched the in-page header in favor of making the `title` metadata consistent, and using that at the layout level to always render it consistently.